src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag

```diff
@@ -0,0 +1,49 @@
+<%@ tag description="Student Message of the day" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Config" %>
+<c:set var="motdUrl" value="<%= Config.inst().STUDENT_MOTD_URL %>" />
+<c:if test="${motdUrl != null && not empty motdUrl}">
```

Comment:
ohh, ok, I'll get to that in a bit, didn't know that it'll check for not null too!